README.md

```diff
@@ -80,18 +157,29 @@ if ( tour ) {
 > One example is available on [Mamba navigator](https://github.com/mamba-org/gator/blob/master/packages/labextension/src/index.ts#L76).
 > Test it on [binder](https://mybinder.org/v2/gh/mamba-org/gator/master?urlpath=lab).
 
-## Install
+## Disabling the User and Default Tours
 
-```bash
-pip install jupyterlab-tour
-```
+If you _only_ wish to see the default _Welcome_ and _Notebook_ tours, or ones
+defined by users, create a [pageConfig.json] and put it in _the right place_, e.g.
+`{sys.prefix}/etc/jupyter/labconfig/pageconfig.json` and add the plugin IDs to
+`disabledExtensions`.
 
-or
+For example, to disable all tours not provided by other extensions:
 
-```bash
-conda install -c conda-forge jupyterlab-tour
+```json
+{
+  "disabledExtensions": {
+    "jupyterlab-tour:user-tours": true,
+    "jupyterlab-tour:default-tours": true,
+  }
+}
 ```
 
```

Comment:
What about adding the alternative with the command line. Something like

---

Alternatively, you can also use the following commands:

```bash
jupyter labextension disable "jupyterlab-tour:user-tours"
jupyter labextension disable "jupyterlab-tour:default-tours"
```

> To enable the extension, use `jupyter labextension enable ...`.

Reply:
good call, 66b0f95